src/settings/BursarExports/apiQuery.js

```diff
@@ -0,0 +1,61 @@
+import { useMutation, useQuery } from 'react-query';
```

Comment:
react-query will likely be replaced with `swr` library. See https://github.com/folio-org/stripes-core/pull/1015. However it will be added in the Iris release, so no action, for now, is needed.

Reply:
yeah, it's pretty easy to replace

Reply:
interesting, what is the reason? react-query looks more mature, and by comparison it has more features. I heard on stripes-architecture meeting that guys want to look at alternative, just to be sure that react-query is the best

Reply:
The main concern that it maintains only one person and `swr` is backed by the company. Let's discuss it on stripes-architecture.

Reply:
it's one concern, to take a look at swr lib, however I haven't heard that it's a reason we shouldn't use react-query.